crates/bevy_text/src/font_atlas_set.rs

```diff
@@ -84,12 +112,18 @@ impl FontAtlasSet {
         self.font_atlases
             .get(&FloatOrd(font_size))
```

Comment:
We can avoid looking up the character index twice by doing this:

```rust
    pub fn get_glyph_atlas_info(&self, font_size: f32, character: char) -> Option<GlyphAtlasInfo> {
        self.font_atlases
            .get(&FloatOrd(font_size))
            .and_then(|font_atlas| {
                font_atlas
                    .iter()
                    .find_map(|atlas| {
                        atlas
                            .get_char_index(character)
                            .map(|char_index| (char_index, atlas.texture_atlas))
                    })
                    .map(|(char_index, texture_atlas)| GlyphAtlasInfo {
                        texture_atlas,
                        char_index,
                    })
            })
    }
```

Reply:
uh that is nicer .. i still get confused at some of those combinations ;)